responsemanager/server.go

```diff
@@ -58,6 +63,17 @@ func (rm *ResponseManager) processUpdate(key responseKey, update gsmsg.GraphSync
 		log.Warnf("received update for non existent request, peer %s, request ID %d", key.p.Pretty(), key.requestID)
 		return
 	}
+
+	_, span := otel.Tracer("graphsync").Start(trace.ContextWithSpan(rm.ctx, response.span), "processUpdate", trace.WithAttributes(
+		attribute.Int("id", int(update.ID())),
+		attribute.Int("priority", int(update.Priority())),
+		attribute.String("root", update.Root().String()),
+		attribute.Bool("isCancel", update.IsCancel()),
```

Comment:
are these needed? getting to this code path requires that isCancel is always false and isUpdate is always true... seems not needed to record a value that is always the same.

Reply:
also, all of these values EXCEPT the extensions should be contained in the parent span. basically, an update request ONLY contains extensions to update the request plus an `id` to identify the request (plus isUpdate to identify it's an update). 

This is actually a pretty confusing part of the current protobuf format:

new, cancel, and update share a struct type: Request

However, Cancel is basically:
```
{
id: id,
isCancel: true
}
```
and update is:
```
{
id: id,
isUpdate: true,
extensions: extensions
}
```

All the other fields are only set for new requests.

Reply:
Only mentioning in case we decide we want to make this clearer, given that we're about to do big protocol changes.

Reply:
OK, all very good to know. What I've done: removed all but `"extensions"` from the `processUpdate` span, also removed `"isCancel"` and `"isUpdate"` from the `request` span.

It looks like we can assume an `"isUpdate"` if we get a `"processUpdate"` span, but I don't know if we can assume an `"isCancel"` as clearly since `abortRequest()` can be called from both `CloseWithNetworkError()` and `CancelResponse()` as well as from the plain `processRequests()` path. I _think_ that an `ipldutil.ContextCancelError` in the error of the span should differentiate the different paths and let you assume that it's an `"isCancel"`.

But also, maybe this is a task for later when we get some tracing closer up to the message layer where we can lay out more specifics about the chatter.